bootstrap.sh

```diff
@@ -7,10 +7,26 @@ pip3 install --upgrade pip
 
 # Installing pipenv, create new env and sync lib with pipfile
 pip3 install pipenv
-cd /vagrant/givitsite/
+
+#navigate to app folder
+if [ "$IN_VAGRANT" ] 
+then
+    cd /vagrant/givitsite/
+    echo "in vagrant"
+else
+    cd givitsite
+    echo "out of vagrant"
+fi
+
 pipenv sync
 
 # DB creation
+if [ -z "$IN_VAGRANT" ] 
```

Comment:
What is the logic behind DB creation when $IN_VAGRANT is not defined?

Reply:
@lioramilbaum if it isn't defined (when running on github actions), it's needed to start postgres before the DB creation

Reply:
Then use the CI environment variable which is defined by GitHub Actions but not defined when vagrant is up.
Today you are using vagrant, tomorrow it might be some other technology. This script would break.